web-marketplace/src/pages/CreditClassDetails/CreditClassDetails.utils.ts

```diff
@@ -95,16 +99,20 @@ export const parseCreditClassVersion = (
 
 type NormalizeImpactInput = {
   impact?: Impact;
+  projectImpact?: ProjectImpact;
   sanityImpact?: EcologicalImpact;
   sdgs?: Maybe<Maybe<Sdg>[]>;
 };
 
 const normalizeImpact = ({
   impact,
+  projectImpact,
   sanityImpact,
   sdgs,
 }: NormalizeImpactInput) => ({
   name: impact?.['schema:name'] || sanityImpact?.name,
+  description:
+    projectImpact?.['schema:description'] || sanityImpact?.descriptionRaw,
```

Comment:
@erikalogie if there's some generic description on sanity for an impact, do we still want to display it on the project page as on https://deploy-preview-2067--regen-marketplace.netlify.app/project/C01-002 or on the credit class page https://deploy-preview-2067--regen-marketplace.netlify.app/credit-classes/C01 ?

Reply:
I don't see anything on the latest design: https://www.figma.com/file/seUiQM2SPt78KJiDL0Ea8m/Basic-Ecocredits-Ledger-View-%2F-Marketplace?node-id=9376%3A250796&mode=dev

Reply:
No we don't want to display the generic description anymore on the project page, just the specific one if it exists in the metadata.